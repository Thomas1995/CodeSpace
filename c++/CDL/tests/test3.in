Testam parsare pe numere negative

Vreau un patrat in punctul (-5,-8) de latura 4
