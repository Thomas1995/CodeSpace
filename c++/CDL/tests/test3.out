0 Patrat:
-5 -8
-5 -4
-1 -8
-1 -4
